okhttp-clients/src/test/java/com/palantir/conjure/java/okhttp/ExponentialBackoffTest.java

```diff
@@ -33,15 +33,15 @@
     @Test
     public void testNoRetry() {
         Random random = mock(Random.class);
```

Comment:
seems a little weird to mock `Random` now that we're just passing it a supplier

Reply:
That's fair, updated to use a simple AtomicDouble.